### ISSUE TITLE
Possibility to access website from multiple teams

### ISSUE THREAD

**Issue by bayerlse** [umami-software/umami#3032]
### Describe the feature or enhancement

In our working environment, we have different teams that need to have access to some websites, that other teams have already access to. Currently I cant enable the following:

team_1:

- website-1
- **website-2**
- website-3

team_2:

- **website-2**
- website-4

Also as admin I would like to see all websites that are created, to have a better overview. 

Thanks!

**Comment by mikecao** [comment]
Can't you just add team members to both teams?

**Comment by bayerlse** [comment]
No, because the first team is just for internals and the second is shared with some members that should not have access to all internal pages.
